blog/config.toml

```diff
@@ -9,6 +9,8 @@ pygmentsStyle = "solarized-dark"
 pygmentsCodefencesGuessSyntax = true
 pygmentsOptions = "linenos=table"
 
+googleAnalytics = "UA-39420295-8"
```

Comment:
@artur-rs by the way added google analytics code, I quess blog wasn't monitored before

Reply:
@artur-rs @lukasz-matysiewicz that looks like a significant failure